packages/bff/src/model/domain/errors.ts

```diff
@@ -6,6 +6,9 @@ import {
 
 export const errorCodes = {
   purposeNotFound: "0001",
+  missingClaim: "9990",
+  unknownTenantOrigin: "0011",
+  invalidJwtClaim: "0000", // Not used, change it if needed for a more specific error
```

Comment:
This comment is a leftover? if yes 🧹

Reply:
No it's not a leftover, in scala there isn't any code.

Reply:
Ok, in this case, for me it can be removed. As policy, during the migration, we have the ability to create new errors and codes.

Reply:
Yes, but since we didn't complete the migration I'm not sure if someone else needs that code, so if needed the comment explain it can be used.

Reply:
we can give all errors here a code.
Since all codes have been re-arranged, you can just use the next available value